.cleanmodules

```diff
@@ -0,0 +1,11 @@
+# common extra inclusions
+**/*.d.ts
+**/@types/**
+**/*.@(yaml|yml)
+licen@(s|c)e*(.*)
```

Comment:
https://github.com/duniul/clean-modules/blob/main/.cleanmodules-default#L148 license is already included